src/main/app/MiningWorkerSemanticConflictSample.groovy

```diff
@@ -0,0 +1,33 @@
+package main.app
+
+import java.text.SimpleDateFormat
+import java.io.File
+
+import main.interfaces.*
+import static main.app.MiningFramework.arguments
+import java.util.concurrent.BlockingQueue
+
+
+import main.arguments.*
+import main.project.*
+import main.interfaces.*
+import main.exception.InvalidArgsException
+import main.exception.UnstagedChangesException
+import main.exception.UnexpectedPostScriptException
+import main.exception.NoAccessKeyException
+import main.util.*
+
+class MiningWorkerSemanticConflictSample extends MiningWorker {
+
+    public MiningWorkerSemanticConflictSample(Set<DataCollector> dataCollectors, CommitFilter commitFilter, BlockingQueue<Project> projectList, String baseDir) {
+        super(dataCollectors, commitFilter, projectList, baseDir)
+    }
+
+    @Override
+    protected void cloneRepository(Project project, String target) {    
```

Comment:
@leusonmario por que a implementacao original nao era adequada para voce? 

ela parece ser mais interessante do que essa.

se for realmente necessario ter duas implementacoes para esse metodo, melhor criar uma classe abstrata MiningWorker, com o método abstrato  cloneRepository e mais tudo da classe atual que for comum aos dois estudos. ela seria herdada por duas classes, uma para o seu estudo e outra para o estudo de @rafaelmotaalves, cada uma com o código especifico do estudo

Reply:
Minha intenção era obter o path local do projeto, e assim setar esse atributo da classe Project. Eu poderia ter feito isso na classe MiningWorker, mas como não estava certo que essa mudança faria sentido para Rafael, acabei optando por criar esta nova classe.